.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -12,7 +12,7 @@ body:
     id: problem
     attributes:
       label: Related problem
-      description: A clear and concise description of what the thing is, that Nushell could do better.
+      description: A clear and concise description of what specifically Nushell could do to improve.
```

Comment:
okay i'm nitpicking but can the "specifically" be italicized, like `*specifically*` lol

Reply:
And I thought, I was passive agressive ^^

Maybe needs to be quoted so YAML doesn't puke but we can try it.